server/views/pages/prison-register/prisonDetails.njk

```diff
@@ -140,7 +148,7 @@
         text: "Prison"
       },
         value: {
-        html: prisonname(prisonDetails.id, prisonDetails.name, prisonDetails.active, prisonDetails.male, prisonDetails.female, prisonDetails.contracted, prisonDetails.types),
```

Comment:
I think at some point we can tidy this method to just pass in prisonDetails.  Not for this ticket tho.